OpenRA.Editor/Form1.cs

```diff
@@ -238,7 +238,9 @@ void PrepareMapResources(ModData modData, Map map)
 					if (rsi != null && rsi.EditorPalette != null && rsi.EditorPalette.Contains("terrain"))
 						templatePalette = palette;
 
-					var template = RenderUtils.RenderActor(info, tileset, templatePalette);
+					var race = Program.Rules.Actors["world"].Traits.WithInterface<CountryInfo>().First().Race;
```

Comment:
As people are reviewing #4496 extremely picky already, I can imagine the first "bug" reports: "I can only place Atreides buildings" etc. This could be solved with a player country/color selector window.